spec/gibbon/gibbon_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper'
 require 'cgi'
+require 'webmock'
+require 'webmock/rspec'
```

Comment:
Is the introduction of webmock necessary because Net::HTTP can't be mocked using rspec like the other API tests?